federationapi/routing/join.go

```diff
@@ -176,9 +176,10 @@ func SendJoin(
 
 	return util.JSONResponse{
 		Code: http.StatusOK,
-		JSON: map[string]interface{}{
-			"state":      stateAndAuthChainRepsonse.StateEvents,
-			"auth_chain": stateAndAuthChainRepsonse.AuthChainEvents,
+		JSON: gomatrixserverlib.RespSendJoin{
+			StateEvents: stateAndAuthChainRepsonse.StateEvents,
```

Comment:
@babolivier 
Please correct the spelling of `stateAndAuthChainRepsonse` to `stateAndAuthChainResponse` in this function.